### ISSUE TITLE
.NET 5 Support

### ISSUE THREAD

**Issue by adamhathcock** [HangfireIO/Hangfire#1766]
Looks like only the dashboard needs to be updated/supported.

Would a PR be accepted for this?

Thanks!

**Comment by odinserj** [comment]
Of course, but some days ago I've tested everything on .NET 5.0 as I remember (was testing Hangfire.InMemory package, not the newest .NET), and didn't get any issues.

**Comment by adamhathcock** [comment]
Tricky for me to test with all of the other targets as I'm not on Windows but I'll try to do something soon, kids allowing, unless someone beats me.

**Comment by sabiland** [comment]
I’ve updated to Net5 few days ago and Hangfire runs without any problems, also dashboard works ok!
